### ISSUE TITLE
How about support Weights & Biases(wandb) Feature Addition?

### ISSUE THREAD

**Issue by piantic** [POSTECH-CVLab/PyTorch-StudioGAN#28]
Hi, 

W&B is a tool to support for debugging models and metric logging like a tensorboard.
Although it is a different filed, Support W&B raised an issue in YoloV5 and W&B was officially added four days ago.

https://wandb.ai/cayush/yoloV5/reports/How-are-your-YOLOv5-models-doing---VmlldzoyNjM3MTY

I think that it is good choice for this repo.
Thanks.

# About Weights & Biases
Think of W&B like GitHub for machine learning models — save everything you need to debug, compare and reproduce your models — architecture, hyperparameters, weights, model predictions, GPU usage, git commits, and even datasets — with a few lines of code.

W&B lightweight integrations work with any Python script, and all you need to do is sign up for a free W&B account to start tracking and visualizing your models.

Used by the likes of OpenAI, Lyft, Github and researchers at top machine learning labs across the world, W&B is part of the new standard of best practices for machine learning.

How W&B can help you optimize your machine learning workflows:

Debug model performance in real time
- Automatically tracked GPU, CPU usage and other system metrics
- Powerful custom charts
- Share model insights interactively
- Efficient hyperparameter optimization
- Dataset and model pipeline tracking and production model management
- W&B is free for individuals, academics and open source projects.
[](https://user-images.githubusercontent.com/15766192/97772129-fe7c3100-1b69-11eb-976e-7fa19fcdb4c5.png)

**Comment by MINGUKKANG** [comment]
Hi:)

Thank you for your suggestion.

Weights and Biases is a nice visualization tool, but I decided not to use it due to a number of issues.

Thank you!

Sincerely,

**Comment by syncle** [comment]
@MINGUKKANG could you elaborate on what are the issues? Maybe good to open a discussion if necessary.

**Comment by MINGUKKANG** [comment]
@syncle 

Hello.

The visualization library (Weights and Biases) is free for personal use, but there is a billing policy for the use of team units and companies:(

So, I decided that I would not use the Weights and Biases.

Thank you!
